mappings/net/minecraft/client/model/Model.mapping

```diff
@@ -1,3 +1,5 @@
 CLASS net/minecraft/class_3879 net/minecraft/client/model/Model
 	FIELD field_17138 textureWidth I
 	FIELD field_17139 textureHeight I
+	METHOD method_22696 onPartAdded (Lnet/minecraft/class_630;)V
```

Comment:
`addModelPart`, as in `PlayerEntityModel` it adds the part to the list?

Reply:
this sounds like a callback instead of straight addition.

Reply:
I found no usages of this method, so it's impossible to tell.

Reply:
It's used in the `ModelPart` constructor. Whenever a model part is created, this callback gets called. It's currently only implemented by the player model to keep the parts in a list (so that it can return random parts for the "sticking out things" feature renderer).